controllers/apps/transformer_component_workload.go

```diff
@@ -388,10 +406,6 @@ func checkNRollbackProtoImages(itsObj, itsProto *workloads.InstanceSet) {
 	for i, cc := range [][]corev1.Container{itsObj.Spec.Template.Spec.InitContainers, itsObj.Spec.Template.Spec.Containers} {
 		images[i] = make(map[string]string)
 		for _, c := range cc {
-			// skip the kb-agent container
```

Comment:
@leon-inf Why skips kb-agent container here in the first place? I deleted this because when registry config changes, kb-agent container image will be replaced to the new registry. IMO this is not the expected behavior.